modules/core/src/main/scala/org/tesselation/domain/aci/StateChannelContext.scala

```diff
@@ -0,0 +1,8 @@
+package org.tesselation.domain.aci
+
+import org.tesselation.schema.address.Address
+
+trait StateChannelContext[F[_]] {
+  val address: Address
+  def createCell(input: Array[Byte]): F[StdCell[F]]
+}
```

Comment:
IMO it shouldn't be in the `core` module. probably `kernel` is a better place for this
@kpudlik @TheMMaciek ?

Reply:
I think that the `kernel` module should contain only the stuff necessary for the users to create the StateChannel. This context is an internal thing, that gets created when loading a jar.

Reply:
but related to state channels 🤔 let's discuss it on slack